client/state/media/reducer.js

```diff
@@ -289,12 +289,16 @@ export const selectedItems = withoutPersistence( ( state = {}, action ) => {
 			};
 		}
 		case MEDIA_ITEM_REQUEST_SUCCESS: {
-			const { mediaId: transientMediaId, siteId } = action;
+			const { mediaId: requestedMediaId, siteId } = action;
```

Comment:
heads up @tyxla and/or @saramarcondes that I changed this line to reflect single non-transient items being requested (like in #44640); as far as I can see transient item selection still works as expected